src/main/java/org/isf/priceslist/rest/PriceListController.java

```diff
@@ -0,0 +1,183 @@
+import org.isf.priceslist.dto.PriceListDTO;
+import org.isf.priceslist.manager.PriceListManager;
+import org.isf.priceslist.mapper.PriceListMapper;
+import org.isf.priceslist.mapper.PriceMapper;
+import org.isf.priceslist.model.Price;
+import org.isf.priceslist.model.PriceList;
+import org.isf.shared.exceptions.OHAPIException;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
+import org.isf.utils.exception.model.OHSeverityLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+
+@RestController
+@Api(value = "/pricelists", produces = MediaType.APPLICATION_JSON_VALUE)
+public class PriceListController {
+
+	@Autowired
+	protected PriceListManager priceListManager;
+	
+	@Autowired
+	protected PriceListMapper mapper;
+	
+	@Autowired
+	protected PriceMapper priceMapper;
+
+	private final Logger logger = LoggerFactory.getLogger(PriceListController.class);
+
+	public PriceListController(PriceListManager priceListManager, PriceListMapper priceListmapper) {
+		this.priceListManager = priceListManager;
+		this.mapper = priceListmapper;
+	}
+
+	/**
+	 * create a new {@link PriceList}
+	 * @param priceListDTO
+	 * @return <code>true</code> if the price list has been stored, <code>false</code> otherwise.
+	 * @throws OHServiceException
+	 */
+	@PostMapping(value = "/pricelists", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<String> newPriceList(@RequestBody PriceListDTO priceListDTO) throws OHServiceException {
+		logger.info("Create price list " + priceListDTO.getCode());
+		boolean isCreated = priceListManager.newList(mapper.map2Model(priceListDTO));
+		if (!isCreated) {
+			throw new OHAPIException(new OHExceptionMessage(null, "price list is not created!", OHSeverityLevel.ERROR));
+		}
+		return ResponseEntity.status(HttpStatus.CREATED).body(priceListDTO.getCode());
+	}
+
+	/**
+	 * Updates the specified {@link PriceList}.
+	 * @param priceListDTO
+	 * @return <code>true</code> if the price list has been updated, <code>false</code> otherwise.
+	 * @throws OHServiceException
+	 */
+	@PutMapping(value = "/pricelists", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<String> updatePriceListt(@RequestBody PriceListDTO priceListDTO)
```

Comment:
Put Method should have code as @PathVariable. I know it should be an http standard.
`@PutMapping(value = "/pricelists/{id}", produces = MediaType.APPLICATION_JSON_VALUE)`